src/main/kotlin/racing/domain/Capacity.kt

```diff
@@ -0,0 +1,23 @@
+package racing.domain
+
+@JvmInline
+value class Capacity(val value: Int) : Iterable<Int>, Comparable<Capacity> {
+    constructor(value: String?) : this(
+        value?.toIntOrNull()
+            ?: throw IllegalArgumentException("Invalid value: $value required number formattable value")
+    )
+
+    init {
+        require(value >= 0) {
+            "Invalid value: $value required positive number"
+        }
+    }
+
+    override fun iterator(): Iterator<Int> = (0 until value).iterator()
+
+    override fun compareTo(other: Capacity): Int = value.compareTo(other.value)
```

Comment:
사소하지만 `compareTo`가 infix 함수인 것을 생각하면 다음과 같이 표현할 수도 있을 것 같아요!
```suggestion
    override fun compareTo(other: Capacity): Int = value compareTo other.value
```